Kernel/RobSubstitution.hpp

```diff
@@ -35,34 +35,35 @@
 #endif
 
 
-const int GLUE_INDEX=-3;
-const int SPECIAL_INDEX=-2;
```

Comment:
No more `SPECIAL_INDEX`? Happy news if so.

Reply:
Yes indeed. How we got around it is a bit technical but the short answer is that one was never needed in the first place, and having it made things less general.